### DISCUSSION TITLE
Add opengraph support

### DISCUSSION THREAD

**Discussion by ssyberg** [payloadcms/payload-plugin-seo#49] [Ideas]
Would be great to allow for opengraph field support here as well!

**Comment by Elliot67** [comment]
Would also be great to have json-ld support.

**Comment by jacobsfletch** [comment]
For now you can define your own custom og or json-ld fields using the `fields` property (fixed in PR https://github.com/payloadcms/plugin-seo/pull/48). Long-term, I _do_ think this is a good idea and we should definitely come up with some permanent solution for this. I'm going to migrate this to a discussion with the "Ideas" category so that we don't lose sight of this feature.
